aws/sdk/examples/lambda-invoke-function/src/main.rs

```diff
@@ -2,52 +2,139 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0.
  */
+ 
+//! Minimal example that invokes an AWS Lambda function
+//!
+//! Invokes a function and prints the response without attempting to
+//! deserialize the returned data.
 
-use std::process;
+/// types from the Rust standard library
+use std::{process, str};
 
+/// types from the AWS SDK for Rust
 use lambda::{error::InvokeErrorKind, Client, Config, Region, SdkError};
-
 use aws_types::region::ProvideRegion;
 
+/// types from other third-party crates
 use tracing_subscriber::fmt::format::FmtSpan;
 use tracing_subscriber::fmt::SubscriberBuilder;
 
 #[tokio::main]
-async fn main() {
+async fn main()  {
+    // We are going to attempt to load the AWS region from the default provider
+    // or else we will default to the us-east-1 region (also referred to as 
```

Comment:
The code has us-west-2 if I am not missed reading.

Reply:
The Lambda function is in us-west-2, and the correct region is being pulled in from my env variables, but this should be defaulting to us-east-1 if I hadn't supplied the region, no?

Reply:
oof, you're right. I am not a SmartMan™. Fixed now.